blues/simulation.py

```diff
@@ -509,11 +616,15 @@ def simulateMD(self, nstepsMD=5000, **opt):
         self.nc_sim.context.setPositions(md_state0['positions'])
         self.nc_sim.context.setVelocities(md_state0['velocities'])
 
-    def run(self, nIter=100):
+    def run(self):
```

Comment:
The way the code is set up right now I don't think you could even set up all the simulations without specifying nIter (ex SimulationFactory expects nIter to be there). 

So rather than confuse the user by allowing two different values of nIter in two different places (the opt dict and simulation.run), I'd rather have it so the behavior is consistent–as my change here does.

Reply:
That's a good point. Can you point to where SimulationFactory expects nIter and if that is even necessary?

Reply:
Actually looking into, it before `run()` nIter is only used in generateSimFromStruct() as an argument, and it's never actually used in the function itself https://github.com/MobleyLab/blues/blob/master/blues/simulation.py#L130,
 so if I'll just remove it from the __init__() there, and just make `nIter` a required argument for `run()`.

So the end result would for `run()` would just be:
```python
def run(nIter):
     #do all the running blues simulation things here
```